content/docs/command-reference/exp/init.md

```diff
@@ -18,11 +18,11 @@ usage: dvc exp init [-h] [-q | -v] [--run] [--interactive] [-f]
 
 ## Description
 
-`dvc exp init` helps you get started with DVC Experiments quickly. It reduces
-boilerplate DVC procedures by creating a `dvc.yaml` file that assumes standard
-locations of your input data, <abbr>parameters</abbr>, source code, models,
-<abbr>metrics</abbr> and [plots](/doc/command-reference/plots). These locations
-can be customized through the [options](#options) below or via
+`dvc exp init` helps you get started with DVC Experiments quickly. It creates an
+experiment pipeline in a dvc.yaml file, which assumes standard locations for the
+inputs (source code, data, and <abbr>parameters</abbr>) and outputs (model,
+<abbr>metrics</abbr>, and <abbr>plots</abbr>) of your model training command.
```

Comment:
@daavoo feel free to add the metrics (and plots if you want) tooltip files in this PR (if you have time soon, otherwise no worries).